### PR TITLE
Update ParkerWeight.m

### PR DESCRIPTION
**Pull request by haozhangbit**
eliminate the requirements: 1. angles must be incremental; 2. angles must start from 0

### CODE REVIEW

File: MATLAB/Utilities/filtering.m [hunk 1]
```diff
@@ -43,9 +44,11 @@
     
     fproj = (real(ifft(fproj)));
     
-  
-    proj(:,:,ii) = fproj(end/2-geo.nDetector(1)/2+1:end/2+geo.nDetector(1)/2,:)/2/geo.dDetector(1)*(2*pi/  size(angles,2)   )/2*(geo.DSD(ii)/geo.DSO(ii));
-    
+    if parker
+   	proj(:,:,ii) = fproj(end/2-geo.nDetector(1)/2+1:end/2+geo.nDetector(1)/2,:)/2/geo.dDetector(1)*(2*pi/  (pi/angle_step)   )/2*(geo.DSD(ii)/geo.DSO(ii));
```
Comment (AnderBiguri) [inline MATLAB/Utilities/filtering.m:48]: Hi! Could you explain this line please? I think I understand why you are doing this, but would it not make sense to get rid of the `2*pi` somehow? Can you share some maths/paper for me to understand this?

File: MATLAB/Utilities/ParkerWeight.m [hunk 2]
```diff
@@ -15,14 +15,16 @@
 %   publisher={American Association of Physicists in Medicine}
 % }
 if unique(geo.DSD)>1
```
Comment (AnderBiguri) [inline MATLAB/Utilities/ParkerWeight.m:17]: There is a big here that was there in my code. This should be `if length(unique(geo.DSD))>1`. Could you add that?

File: MATLAB/Utilities/filtering.m (hunk 1 above)
Comment (haozhangbit) [reply MATLAB/Utilities/filtering.m:48]: Hi Ander, I referred to code by Kyungsang Kim (https://www.mathworks.com/matlabcentral/fileexchange/35548-3d-cone-beam-ct-cbct-projection-backprojection-fdk-iterative-reconstruction-matlab-examples), and do not fully understand the math yet. I will try to figure it out later. 

BTW, which reference did you use for the filtering weighting for non-Parker one? 
proj(:,:,ii) = fproj(end/2-geo.nDetector(1)/2+1:end/2+geo.nDetector(1)/2,:)/2/geo.dDetector(1)*(2*pi/  size(angles,2)   )/2*(geo.DSD(ii)/geo.DSO(ii));

Comment (AnderBiguri) [reply MATLAB/Utilities/filtering.m:48]: I actually took it from there! Ah, I removed the parker apparently  (this was over 6 years ago).
My bad!

**Comment by AnderBiguri** [comment]
This Resolves #104

**Pull request merged**